log.go

```diff
@@ -0,0 +1,46 @@
+package authenticated_block_store
+
+import (
+	"fmt"
+)
+
+type LogManager struct{
+        fileLimit int
+}
+
+// Table API, called by: package.api()
```

Comment:
Is the constructor (below) really the Table API? Might want to update this comment.

Also, caps/complete sentences in comments.